src/Grids/vertically_stretched_rectilinear_grid.jl

```diff
@@ -323,6 +323,29 @@ all_z_nodes(::Type{Face}, grid::VerticallyStretchedRectilinearGrid) = grid.zᵃ
 # Get minima of grid
 #
 
-min_Δx(grid::VerticallyStretchedRectilinearGrid) = grid.Δx
-min_Δy(grid::VerticallyStretchedRectilinearGrid) = grid.Δy
-min_Δz(grid::VerticallyStretchedRectilinearGrid) = minimum(view(grid.Δzᵃᵃᶜ, 1:grid.Nz))
+function min_Δx(grid::VerticallyStretchedRectilinearGrid)
```

Comment:
These three functions are identical to those in `regular_rectilinear_grid.jl` except that they act on different grids.  I wonder if we can merge these to one set of functions that act on either grid?

Reply:
I agree this is a bit verbose, but I'd argue that it makes things much clearer. My attempts at unifying those ended up much less readable.

Reply:
Probably good to get something working sooner rather than later and then someone try and polish things off, when so inspired.